src/main/scala/nl/biopet/bioconda/BiocondaPlugin.scala

```diff
@@ -162,51 +189,94 @@ object BiocondaPlugin extends AutoPlugin {
   }
 
   /**
-    * A task that returns a sequence of net yet published tags and the latest tag
-    * @return sequence of not yet published tags, latest tag
+    * Gets the released tags from github and determines the latest
+    * @return the latest to be released tag.
     */
-  private def getUnPublishedTags: Def.Initialize[Task[R]] = {
+  private def getLatestTag: Def.Initialize[Task[TagName]] = {
     Def.task {
-      val publishedTags: Seq[TagName] = getPublishedTags.value
-      val releasedTags: Seq[TagName] = getReleasedTags.value
-      val latestTag = releasedTags
+      getReleasedTags.value
         .sortBy(tag => tag.stripPrefix("v"))
```

Comment:
This sort is based on strings and not on numbers. Common Utils does have some method to parse a version number like 0.3.6 into a case class. This was by calling just sorted already works